pegasus/sites.v3/code.org/homepage.json

```diff
@@ -116,6 +117,30 @@
           "img_src": "/images/banners/banner-curriculum-catalog-illustration-black.png"
         }
       ]
+    },
+    "ai-pl-launch-2023": {
+      "dcdo": "ai-pl-launch-banners",
```

Comment:
Does setting this dcdo variable here put it behind that flag?

Reply:
Yep!

Reply:
Cool, TIL!